neuralmonkey/nn/projection.py

```diff
@@ -0,0 +1,47 @@
+"""
+This module implements various types of projections.
+"""
+#tests: lint
+import tensorflow as tf
+
+def linear(inputs, size, scope="LinearProjection"):
```

Comment:
It seems to be almost the same as function `nn.mlp.dense`. We should keep only one of them, probably this one and add maybe add an optional non-linearity here.